.github/workflows/pull-request-reviewer-reminder.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Pull request reviewer reminder'
+on:
+  schedule:
+    # Check reviews every weekday, 10:00 and 17:00
+    - cron: '0 10,17 * * 1-5'
+    
+jobs:
+  pull-request-reviewer-reminder: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tommykw/pull-request-reviewer-reminder-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }} # Required
+          reminder_message: 'Two business days have passed since the review started. Pretty please review the PR?' # Required. Messages to send to reviewers on Github.
```

Comment:
```suggestion
          reminder_message: 'Two whole business days have passed since the review started. Pretty please review the PR.' # Required. Messages to send to reviewers on Github.
```

Reply:
1.25 days actually right?